corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -124,32 +129,54 @@ def handle(self, app_name, **options):
                     show_apply_commit=not has_changes
                 )
 
+        self.no_split = options.get('no_split')
         self.skip_all = options.get('skip_all')
+        if self.skip_all and self.no_split:
+            self.stderr.write(
+                "\n--skip-all and --no-split cannot be used at the same time.\n"
+            )
+            return
+
+        if self.skip_all and (js_name or template_name):
+            self.stderr.write(
+                "\n--skip-all cannot be used with --template-name or --js-name\n"
+            )
+            return
+
         if self.skip_all:
             confirm = get_confirmation("You have elected to skip all the confirmation prompts. "
                                        "Are you sure?")
             if not confirm:
                 return
+
         if self.skip_all and has_pending_git_changes():
             self.stdout.write(self.style.ERROR(
                 "You have un-committed changes. Please commit these changes before proceeding...\n"
             ))
             ensure_no_pending_changes_before_continuing()
 
+        if self.no_split:
```

Comment:
I don't think this is necessary. This isn't a dangerous or un-revert-able operation.

Reply:
removed, as well as the prompt for `self.skip_all`